images/linux/scripts/SoftwareReport/SoftwareReport.Common.psm1

```diff
@@ -331,6 +331,20 @@ function Get-DotNetCoreSdkVersions {
     return $dotNetCoreSdkVersion
 }
 
+function Get-DotnetTools {
+    $dotnetToolset = (Get-ToolsetContent).dotnet
+    $dotnetTools = $dotnetToolset.tools
+
+    $toolsList = @()
+
+    ForEach ($dotnetTool in $dotnetTools) {
+        $version = $dotnetTool.getversion
+        $toolsList += [PSCustomObject]@{
+            ToolInfo = "$dotnetTool.name v" + $version
```

Comment:
$dotnetTool.name -> $($dotnetTool.name) or use $dotnetTool.name + " v" + $version

Reply:
Of course!